src/main/java/uk/gov/justice/digital/nomis/jpa/entity/CaseloadAgencyLocation.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.nomis.jpa.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "CASELOAD_AGENCY_LOCATIONS")
+@Data()
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@EqualsAndHashCode(of = {"caseload", "agencyLocation"})
+@ToString(of = {"caseload", "agencyLocation", "updateAllowedFlag"})
+public class CaseloadAgencyLocation implements Serializable {
+
+    @EmbeddedId
+    private CaseloadAgencyIdentity id;
+
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
Why explicitly lazy on a one to one?

Reply:
I like to make every lazy - prevents joins on big query selects